src/platform-includes/enriching-events/breadcrumbs/automatic-breadcrumbs/react-native.mdx

```diff
@@ -0,0 +1,14 @@
+The React Native SDK captures breadcrumbs automatically using the native Android and iOS SDKs:
+
+- [Automatic breadcrumbs for Android](/platforms/android/enriching-events/breadcrumbs/#automatic-breadcrumbs)
+- [Automatic breadcrumbs for iOS](/platforms/apple/configuration/integrations/default/#sentryautobreadcrumbtrackingintegration)
+
+More breadcrumbs are captured by the React Native SDK:
```

Comment:
Should this maybe be, "can be captured" since you still have to take some action to have these breadcrumbs captured?
```suggestion
More breadcrumbs are captured by the React Native SDK:
```

Reply:
Yes, I totally agree. Thank you.